index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tasnimul Hasan Portfolio</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+
+</head>
+<body>
+
+    <header>
+        <div class="nav-menu-content">
+
+            <div class="logo"><a class=mylogo> My Logo </a></div>
+
+            <nav>
+                <ul class="nav-menu">
+                    <li class="nav-items"><a href="#">Portfolio</a></li>
+                    <li class="nav-items"><a href="#">About</a></li>
+                    <li class="nav-items"><a href="#">Contact</a></li>
+                  </ul>
+
+
+                    <div class="hamburger">
+
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                    </div>
+            </nav>
+        </div>
+
+    </header>
+    <br>
+    <br>
+    <section class="homepage">
+        <div class="introduction">
+            <div class="intro"><h1>I'm Tasnimul <br>Glad to meet you!</h1></div>
+            <br>
+            <br>
+            <br>
```

Comment:
### OPTIONAL

- [ ] Kindly remove all br tags and use margin and padding(top, bottom, left, right) to make your design look like the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) this also helps you understand the [css box model](https://www.w3schools.com/css/css_boxmodel.asp) 👍🏿